packages/runtime/src/errors/errors-list.ts

```diff
@@ -8,297 +9,309 @@ export interface ErrorDescriptor {
   description: string
 }
 
-export function getErrorCode (error: ErrorDescriptor): string {
-  return `${ERROR_PREFIX}${error.number}`;
+export function getTealErrorCode (error: ErrorDescriptor): string {
+  return `${TEAL_ERROR_PREFIX}${error.number}`;
 }
 
-export const ERROR_RANGES = {
-  TEAL: { min: 0, max: 99, title: "TEAL opcode errors" },
-  TRANSACTION: { min: 100, max: 199, title: "Transaction error" },
-  ASA: { min: 900, max: 999, title: "ASA File Parse Error" }
+export function getRuntimeErrorCode (error: ErrorDescriptor): string {
+  return `${RUNTIME_ERROR_PREFIX}${error.number}`;
+}
+
+export const TEAL_ERROR_RANGES = {
+  TEAL: { min: 1000, max: 1099, title: "TEAL opcode errors" }
+};
+
+export const RUNTIME_ERROR_RANGES = {
+  GENERAL: { min: 1300, max: 1399, title: "Runtime General Error" },
+  TRANSACTION: { min: 1400, max: 1499, title: "Transaction error" },
+  ASA: { min: 1500, max: 1599, title: "ASA Error" }
```

Comment:
Shouldn't `TEAL` error be also part of `RUNTIME_ERRORS` ? `Runtime` is a superset which does TEAL execution while processing transactions

Reply:
Yes good idea, so as you suggest removing teal errors, and making it as runtime errors with teal error as subset ?
What do you think ?

Reply:
But that will result in all errors in one class, adding more errors will be confusing, and if we want to import them in algob that will be not good also

Reply:
adding it as subset so there will be different classes for both